src/components/ReviewList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import ReviewCard from "./ReviewCard";
+import { getReviews } from "../api";
+
+const ReviewList = ({ category }) => {
+  const [reviews, setReviews] = useState([]);
+  useEffect(() => {
+    getReviews().then(({ reviews }) => {
```

Comment:
Could you add some loading state to let the user know that something is happening,, whilst the data is being fetched?